packages/example-getting-started/docs/controller.md

```diff
@@ -94,29 +93,36 @@ verbs:
 
 #### src/controllers/todo.controller.ts
 ```ts
-import {post, param, requestBody, get, put, patch, del} from '@loopback/openapi-v3';
-import {HttpErrors} from '@loopback/rest';
-import {Todo} from '../models';
 import {repository} from '@loopback/repository';
-import {TodoRepository} from '../repositories/index';
+import {TodoRepository} from '../repositories';
+import {Todo} from '../models';
+import {
+  HttpErrors,
+  post,
+  param,
+  requestBody,
+  get,
+  put,
+  patch,
+  del,
+} from '@loopback/rest';
 
 export class TodoController {
   constructor(
     @repository(TodoRepository.name) protected todoRepo: TodoRepository,
   ) {}
 
   @post('/todo')
-  async createTodo(@requestBody()todo: Todo) {
+  async createTodo(@requestBody() todo: Todo) {
     if (!todo.title) {
       return Promise.reject(new HttpErrors.BadRequest('title is required'));
     }
     return await this.todoRepo.create(todo);
   }
 
   @get('/todo/{id}')
-  async findTodoById(
-    @param.path.number('id') id: number, 
-    @param.query.boolean('items') items?: boolean): Promise<Todo> {
+  async findTodoById(@param.path.number('id') id: number): Promise<Todo> {
+    id = +id;
```

Comment:
Why is this needed?

Reply:
```
// REST adapter does not coerce parameter values coming from string sources
    // like path & query. As a workaround, we have to cast the value to a number
    // ourselves.
    // See https://github.com/strongloop/loopback-next/issues/750
    id = +id;
```

That's from the source code. I'll update the instructions to include a note on that as well.

Reply:
Can you leave a comment on #750 so that it can be tracked and removed once we do have validation?